doc/time/zone.rdoc

```diff
@@ -0,0 +1,7 @@
+- +zone+: a timezone, which may be:
+  - A string offset from UTC.
+  - A single letter offset from UTC, in the range <tt>'A'..'Z'</tt>,
+    <tt>'J'</tt> (the so-called military timezone) excluded.
```

Comment:
```suggestion
    <tt>'J'</tt> excluded (the so-called military timezone).
```

Reply:
I think the grammar is awkward in both cases, I suggest something like:

```suggestion
    <tt>'J'</tt> (the so-called military timezone) is excluded.
```